cli/tests/integration_test.rs

```diff
@@ -47,13 +50,67 @@ mod tests {
             .assert();
         let bytes = assertion.get_output().stdout.as_slice();
         let output = str::from_utf8(bytes)?;
+        assert!(output.contains("-p"));
         assert!(output.contains("--port"));
         assert!(output.contains("The port to run [default: 3000]"));
+        assert!(output.contains("-v"));
         assert!(output.contains("--verbose"));
         assert!(output.contains("Verbose output"));
+        assert!(output.contains("-d"));
+        assert!(output.contains("--detach"));
+        assert!(output.contains("Run in detach mode"));
         Ok(())
     }
 
+    // @todo #129:35min Find a way to run slow tests separately from fast tests.
+    //  This test `accepts_request_in_detached_mode` runs a way longer than
+    //  other unit tests. Let's mark such long tests as slow and run them
+    //  separately from fast test. Locally, developers will run only fast
+    //  tests, while CI server will run both. Check
+    //  <a href="https://www.yegor256.com/2023/08/22/fast-vs-deep-testing.html">this link<a>
+    //  for more information about this idea.
+    #[tokio::test]
+    #[cfg(not(target_os = "windows"))]
+    // @todo #129:60min Create similar integration test for windows platform.
+    //  Now we have test for linux and macos. However, we need to maintain
+    //  similar test case for windows as well.
+    async fn accepts_request_in_detached_mode() -> Result<()> {
+        let _kill = defer(|| kill(3000));
```

Comment:
@h1alexbel i think this name will be more convenient
```suggestion
        let _defer = defer(|| kill(3000));
```